src/components/donate/Contact.js

```diff
@@ -1,7 +1,30 @@
 import React from "react";
 
 const Contact = () => {
-  return <div>Contact</div>;
+  return (
+    <div>
```

Comment:
make sure everything is centered. The whole component will be in the middle of the page. Right now you have it aligned to the left.